frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -49,9 +47,32 @@ const ethnicGroups = [
     label: "Native Hawaiian or Other Pacific Islander",
   },
   { value: "White", label: "White" },
-  { value: null, label: "I prefer not to state" },
+  { label: "I prefer not to state" },
 ];
 
+const customStyles = (theme) => ({
+  icon: {
+    color: "white",
+  },
+});
+
+const CustomTextFieldSelect = withStyles(customStyles)((props) => {
+  const { classes, SelectProps, ...other } = props;
+  return (
+    <TextField
+      select
+      SelectProps={{
+        classes: {
+          icon: classes.icon,
+        },
+        displayEmpty: true,
+        ...SelectProps,
+      }}
+      {...other}
+    ></TextField>
+  );
+});
+
```

Comment:
Let's make this a proper component in the `components` folder, we'll probably need it if other forms pop up

Reply:
Oh and let's just call it `<Select>`! No need for the overly specific name, it's normal to have custom components (the weird and concerning thing is actually this project having so very few) 😄